CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,74 @@
+# Contributor Covenant Code of Conduct
```

Comment:
maybe we should actually put this on our wiki - and add a link to this in all repos - and any wiki concerning PR review process?

Reply:
Good call